New conflict resolution

-  symplify/easy-coding-standard v8.1.7
```
[ERROR] Checkers "PHP_CodeSniffer\Standards\PSR12\Sniffs\Files\FileHeaderSniff" and                                    
         "PhpCsFixer\Fixer\Phpdoc\NoBlankLinesAfterPhpdocFixer" mutually exclude each other. Use only one or exclude the
         unwanted one in "parameters > skip" in your config.
```

--- a/config/config.yaml
+++ b/config/config.yaml
@@ -39,3 +39,4 @@ parameters:
     PhpCsFixer\Fixer\PhpUnit\PhpUnitInternalClassFixer: ~
     PhpCsFixer\Fixer\Phpdoc\NoSuperfluousPhpdocTagsFixer: ~
     SlevomatCodingStandard\Sniffs\Commenting\DisallowCommentAfterCodeSniff.DisallowedCommentAfterCode: ~
+    PHP_CodeSniffer\Standards\PSR12\Sniffs\Files\FileHeaderSniff: ~

